data/campaigns/The_South_Guard/scenarios/01_Born_to_the_Banner.cfg

```diff
@@ -652,6 +655,27 @@
             animate=yes
         [/kill]
 
+        [if]
+            [not]
+                [have_unit]
+                    id="Sir Gerrick"
+                [/have_unit]
+            [/not]
+            [then]
+                {SIR_GERRICK}
```

Comment:
I see... I did notice the scenario_end version too. I think having him actually appear on the map and say something is nicer, though, rather than having a unit suddenly appear in the recall list that you didn't know you had.